src/main/java/uk/gov/hmcts/ccd/domain/model/common/CatalogueResponseGroup.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.ccd.domain.model.common;
+
+public enum CatalogueResponseGroup {
+
+    SUCCESS("00"),
+    VALIDATION("01"),
+    CALLBACK("02");
+
+    final String code;
+
+    CatalogueResponseGroup(final String code) {
+        this.code = code;
+    }
+
+    public String getCode() {
+        return String.format("CCD.%s", code);
```

Comment:
CCD part should also be an attribute. That may help us later in processing similar structure messages from different domains. Let's name it like "domain".